server/db/db-data.js

```diff
@@ -0,0 +1,124 @@
+const mathRandom = function getRandomInt(minParam, maxParam) {
+  const min = Math.ceil(minParam);
+  const max = Math.floor(maxParam);
+
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+const authors = function pickAuthors() {
+  const names = ['Gillian C.', 'Chapman U.', 'Isaiah L.', 'Blair B.', 'Bentle P.', 'Menzie F.', 'Alexander R.', 'Walton S.', 'Rosemary D.', 'Wheeler I.', 'Britney P.', 'Morton M.', 'Damian E.', 'Duncan B.', 'Stan E.', 'Brooks A.', 'Ward O.', 'Benson T.', 'Ferris S.', 'Mitchell E.', 'Ulderico O.', 'Toscani Y.', 'Fiorenza B.', 'Lombardi V.', 'Amando X.'];
+  const index = mathRandom(0, 24);
```

Comment:
Recommend using .length instead so you can modify your arrays without worrying about modifying the range.